cmd/puppeth/wizard_genesis.go

```diff
@@ -73,6 +73,15 @@ func (w *wizard) makeGenesis() {
 			AllConfigs: make(map[uint64]*params.V2Config),
 		},
 	}
+
+	fmt.Println()
+	fmt.Println("What's the name of the subnet chain? (default = xdc-subnet)")
+	genesis.Config.XDPoS.NetworkName = w.readDefaultString("xdc-subnet")
+
+	fmt.Println()
+	fmt.Println("What's the name of the chain denomination? (default = 0x)")
```

Comment:
should we put default as `xdc`?

Reply:
i been thinking to move away from xdc if we can. this caused so many issues. 
But for denom, it doesn't really matter as it's just providing a metadata information. it doesn't do anything, at least for the time being